<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LicenseGenerationClass</class>
 <widget class="QMainWindow" name="LicenseGenerationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LicenseGeneration</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="horizontalSpacing">
     <number>20</number>
    </property>
    <property name="verticalSpacing">
     <number>12</number>
    </property>
    <item row="3" column="2">
     <widget class="QWidget" name="email" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="emailLine"/>
       </item>
       <item>
        <widget class="QLabel" name="lblEmailWarning">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">color : rgb(255, 0, 0)</string>
         </property>
         <property name="text">
          <string>Die eingegebene Email-Adresse ist nicht korrekt!</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QWidget" name="lastName" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="lastNameLine"/>
       </item>
       <item>
        <widget class="QLabel" name="lblLastNameWarning">
         <property name="styleSheet">
          <string notr="true">color : rgb(255, 0, 0)</string>
         </property>
         <property name="text">
          <string>Bitte geben Sie einen Nachnamen an!</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QWidget" name="firstName" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="firstNameLine"/>
       </item>
       <item>
        <widget class="QLabel" name="lblFirstNameWarning">
         <property name="styleSheet">
          <string notr="true">color : rgb(255, 0, 0)</string>
         </property>
         <property name="text">
          <string>Bitte geben Sie einen Vornamen an!</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="11" column="2">
     <widget class="QWidget" name="time" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QComboBox" name="durationCombo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblExpirationDate">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Resultierendes Ablaufdatum: </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QWidget" name="mac" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="macLine"/>
       </item>
       <item>
        <widget class="QLabel" name="lblMacWarning">
         <property name="styleSheet">
          <string notr="true">color : rgb(255, 0, 0)</string>
         </property>
         <property name="text">
          <string>Bitte geben Sie eine vollst√§ndige MAC-Adresse an!</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QWidget" name="company" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="companyLine"/>
       </item>
       <item>
        <widget class="QLabel" name="lblCompanyWarning">
         <property name="styleSheet">
          <string notr="true">color : rgb(255, 0, 0)</string>
         </property>
         <property name="text">
          <string>Bitte geben Sie eine Firma an! Wenn Sie die Lizenz privat nutzen wollen, geben Sie 'privat' an!</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QWidget" name="features" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="featureFullScreenBox">
         <property name="text">
          <string>Vollbildmodus</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="featureSpeedBox">
         <property name="text">
          <string>Abspielgeschwindigkeit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="featureColorBox">
         <property name="text">
          <string>Videofarben</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="featureHistogramBox">
         <property name="text">
          <string>Histogramme</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblFeatureWarning">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 0, 0);</string>
         </property>
         <property name="text">
          <string>Bitte waehlen Sie minderstens ein Feature aus!</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lblLastName">
      <property name="text">
       <string>Nachname:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="16" column="2">
     <widget class="QLabel" name="lblEnterCustNumb">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Placeholder</string>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QLabel" name="lblCustomerNumber">
      <property name="text">
       <string>Kundennummer:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="lblMac">
      <property name="text">
       <string>MAC-Adresse:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lblEmail">
      <property name="text">
       <string>Email-Adresse:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblFirstName">
      <property name="text">
       <string>Vorname:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="lblCompany">
      <property name="text">
       <string>Firma:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="lblFeatures">
      <property name="text">
       <string>Features:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="lblDuration">
      <property name="text">
       <string>Lizenzdauer:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>firstNameLine</tabstop>
  <tabstop>lastNameLine</tabstop>
  <tabstop>emailLine</tabstop>
  <tabstop>companyLine</tabstop>
  <tabstop>macLine</tabstop>
  <tabstop>featureFullScreenBox</tabstop>
  <tabstop>featureSpeedBox</tabstop>
  <tabstop>featureColorBox</tabstop>
  <tabstop>featureHistogramBox</tabstop>
  <tabstop>durationCombo</tabstop>
 </tabstops>
 <resources>
  <include location="LicenseGeneration.qrc"/>
 </resources>
 <connections/>
</ui>
